src/ui/simulator/cmake/windows-calendar.cmake

```diff
@@ -8,4 +8,4 @@ set(SRC_UI_WINDOWS_CALENDAR
 add_Library(antares-ui-windows-calendar ${SRC_UI_WINDOWS_CALENDAR})
 
 target_link_libraries(antares-ui-windows-calendar PRIVATE ${wxWidgets_LIBRARIES})
-target_link_libraries(antares-ui-windows-calendar PRIVATE antares-ui-common)
+target_link_libraries(antares-ui-windows-calendar PRIVATE antares-ui-common Antares::resources Antares::locator)
```

Comment:
why link to **locator** here ? Seems useless

Reply:
https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/1773#discussion_r1476110228